cmd/terramate/cli/cli.go

```diff
@@ -247,7 +247,7 @@ func (c *cli) initStack(dirs []string) error {
 	}
 
 	if len(errmsgs) > 0 {
-		return fmt.Errorf("%w: %v", ErrInit, strings.Join(errmsgs, ": "))
+		return ErrInit
```

Comment:
this change is to avoid a duplicate log message:

```
$ terramate init modules/
warn: failed to initialize stack: directory "/tmp/repo/modules" is inside stack "/tmp/repo" but nested stacks are disallowed
2021/12/15 11:07:52 failed to initialize all stacks: directory "/tmp/repo/modules" is inside stack "/tmp/repo" but nested stacks are disallowed
```

now it shows:

```
$ terramate init modules/
warn: failed to initialize stack: directory "/tmp/repo/modules" is inside stack "/tmp/repo" but nested stacks are disallowed
2021/12/15 11:09:09 failed to initialize all stacks
```